### ISSUE TITLE
Error: cannot enable pretty print when stream specified

### ISSUE THREAD

**Issue by QuantumDream** [pinojs/hapi-pino#27]
Hello guys, maybe I'm missing something but when I follow the example I get the error speficied in the title, my index.js looks like this:

```
const Hapi = require('hapi');
const HapiPino = require('hapi-pino');

async function start() {
  const server = Hapi.server({
    port: process.env.PORT
  });

  await server.register({
    plugin: HapiPino,
    options: {
      prettyPrint: true
    }
  });

  await server.start();
}

start().catch(err => {
  // eslint-disable-next-line
  console.error(err);
  process.exit(1);
});
```

This is the full error in the console (I have shortened the paths):

```
Error: cannot enable pretty print when stream specified
    at pino (node_modules\pino\pino.js:276:42)
    at Object.register (node_modules\hapi-pino\index.js:38:14)
    at internals.Server.register (node_modules\hapi\lib\server.js:399:35)
    at start (index.js:18:16)
    at Object.<anonymous> (index.js:7:1)
    at Module._compile (module.js:635:30)
    at Object.Module._extensions..js (module.js:646:10)
    at Module.load (module.js:554:32)
    at tryModuleLoad (module.js:497:12)
    at Function.Module._load (module.js:489:3)
    at Function.Module.runMain (module.js:676:10)
    at startup (bootstrap_node.js:187:16)
    at bootstrap_node.js:608:3
```

I'm using node@8.9.1, hapi@17.0.1 and hapi-pino@3.0.0, can you maybe help me here? I tried specifying the stream again in the options but that didn't help.

**Comment by davidmarkclements** [comment]
Hapi 17 is not backwards compatible - it's likely that hapi-pino needs to be updated to work with the new hapi

**Comment by mcollina** [comment]
hapi-pino@3.0.0 is compatible with Hapi v17, @davidmarkclements

**Comment by mcollina** [comment]
Fixed in 3.0.1
